istio-cni/values.yaml

```diff
@@ -0,0 +1,20 @@
+hub: gcr.io/istio-release
```

Comment:
There is a bug on all components we are trying to fix - as we refactored to top-level templates we broke backward compat with 1.1 values.yaml, where settings were prefixed by the component name.

Should be:
cni:
   hub: ...

and in templates {{ .Values.cni.hub }}